src/Controller/SantaController.php

```diff
@@ -127,13 +127,28 @@ public function message(Rudolph $rudolph, Request $request, string $application)
         $selectedUsers = $session->get('selected-users', []);
         $message = $session->get('message');
         $notes = $session->get('notes');
+        $options = $session->get('options');
 
         $errors = [];
 
         if ($request->isMethod('POST')) {
             $message = trim($request->request->get('message'));
             $notes = array_map('trim', $request->request->all('notes'));
 
+            $options = [];
```

Comment:
The other options are going to be erased?

Maybe the line 130 should be:

```php
$options = $session->get('options', []);
```

And this one removed.